scripts/skills/perks/perk_rf_swordmaster_metzger.nut

```diff
@@ -16,7 +16,8 @@ this.perk_rf_swordmaster_metzger <- ::inherit("scripts/skills/perks/perk_rf_swor
 		local ret = this.perk_rf_swordmaster_abstract.isEnabled();
 		if (ret)
 		{
-			if (!this.getContainer().getActor().getMainhandItem().isItemType(::Const.Items.ItemType.RF_Southern))
+			local weapon = this.getContainer().getActor().getMainhandItem();
+			if (!weapon.isItemType(::Const.Items.ItemType.RF_Southern) && weapon.isWeaponType(::Const.Items.WeaponType.Cleaver))
```

Comment:
The perk description mentions that it requires a southern sword ...
But a pure northern sword will pass this check because it is not a Cleaver.

This function currently only checks against Northern Cleavers (that are by extensions also swords because of the previous check)
Apart from norther swords above this condition will not check against other hybrid sword types like goblin spear/swords or sword/daggers

Reply:
In the change in this PR, this perk requires a Southern Sword or a Sword/Cleaver hybrid. So the condition says:

So, if the thing is a southern sword, then this `if` will fail. But if the thing _is_ a southern sword, then it checks if it is a cleaver. If it is a cleaver that means it is a Sword/Cleaver hybrid (because `ret` can only be true for swords).

So to me the condition says, either be a southern sword, or be a cleaver (i.e. sword/cleaver hybrid). Which seems correct. 

Lemme know if there's a mistake in my thinking.